frontend/src/components/Teams/CreateTeam/ListMembers/index.tsx

```diff
@@ -99,40 +84,34 @@ const ListMembers = ({ isOpen, setIsOpen }: Props) => {
 							</StyledDialogCloseButton>
 						</DialogClose>
 					</StyledDialogTitle>
-					<FormProvider {...methods}>
-						<Flex css={{ padding: '$24 $32 $40' }} direction="column" gap={16}>
-							<Input
-								currentValue={searchMember}
-								icon="search"
-								iconPosition="left"
-								id="search"
-								maxChars="30"
-								placeholder="Search member"
-								state="default"
-								type="text"
-							/>
-						</Flex>
-						<Text css={{ display: 'block', px: '$32', py: '$10' }} heading="4">
-							Teams
-						</Text>
-						<ButtonsContainer gap={24} justify="end">
-							<Button
-								css={{ margin: '0 $24 0 auto', padding: '$16 $24' }}
-								variant="primaryOutline"
-								onClick={handleClose}
-							>
-								Cancel
-							</Button>
-							<Button
-								css={{ marginRight: '$32', padding: '$16 $24' }}
-								ref={submitBtnRef}
-								type="submit"
-								variant="primary"
-							>
-								Save
-							</Button>
-						</ButtonsContainer>
-					</FormProvider>
+
+					<Flex css={{ padding: '$24 $32 $40' }} direction="column" gap={16}>
+						<InputSearch
+							currentValue={searchMember}
+							icon="search"
+							iconPosition="left"
+							id="search"
+							placeholder="Search member"
+							handleChange={(event: React.ChangeEvent<HTMLInputElement>) =>
+								setSearchMember(event.target.value)
+							}
```

Comment:
I think it is a best practice to use a function reference instead of declaring the function inside the `handle change`
```suggestion
							handleChange={handleSearchChange}
```

```
const handleSearchChange = (event: React.ChangeEvent<HTMLInputElement>) => {
   setSearchMember(event.target.value)
}
```